cmake/scripts/linux/Install.cmake

```diff
@@ -127,6 +127,17 @@ install(FILES ${CMAKE_SOURCE_DIR}/tools/Linux/packaging/media/icon256x256.png
         DESTINATION ${datarootdir}/icons/hicolor/256x256/apps
         COMPONENT kodi)
 
+# Install firewalld service definitions
+install(FILES ${CMAKE_SOURCE_DIR}/tools/Linux/firewalld-services/kodi-eventserver.xml
+        DESTINATION ${prefix}/lib/firewalld/services
+        COMPONENT kodi)
+install(FILES ${CMAKE_SOURCE_DIR}/tools/Linux/firewalld-services/kodi-http.xml
+        DESTINATION ${prefix}/lib/firewalld/services
+        COMPONENT kodi)
+install(FILES ${CMAKE_SOURCE_DIR}/tools/Linux/firewalld-services/kodi-jsonrpc.xml
+        DESTINATION ${prefix}/lib/firewalld/services
+        COMPONENT kodi)
+
```

Comment:
Thank you for your first contribution to Kodi. Very much appreciated.

You can make it shorter:
```cmake
# Install firewalld service definitions
install(FILES ${CMAKE_SOURCE_DIR}/tools/Linux/firewalld-services/kodi-eventserver.xml
              ${CMAKE_SOURCE_DIR}/tools/Linux/firewalld-services/kodi-http.xml
              ${CMAKE_SOURCE_DIR}/tools/Linux/firewalld-services/kodi-jsonrpc.xml
        DESTINATION ${prefix}/lib/firewalld/services
        COMPONENT kodi)
```

Apart from that, looks good to me. Build failure is unrelated to this change.